docs/source/release_notes.rst

```diff
@@ -29,6 +29,7 @@ Release Notes
         * Fix warnings thrown during AutoMLSearch in ``HighVarianceCVDataCheck`` :pr:`1346`
         * Fixed bug where TrainingValidationSplit would return invalid location indices for dataframes with a custom index :pr:`1348`
         * Fixed bug where the AutoMLSearch ``random_state`` was not being passed to the created pipelines :pr:`1321`
+        * Removed pandas warning when converting to Woodwork in ``search()`` :pr:`1358`
```

Comment:
I can't reproduce this with pandas 1.1.1 and woodwork 0.0.3. 

I vote with trying to filter the warning rather than copying the dataframe if possible but this would be ok to merge since the woodwork team is fixing the issue.

Reply:
@freddyaboulton I'm not sure there is a solution for this inside of Woodwork that does not involve copying the dataframe during DataTable initialization. There is an open issue for this that we can continue to investigate, but I'm not sure what can be done at this point.

Some additional comments can be found in the [Woodwork Issue](https://github.com/FeatureLabs/woodwork/issues/312#issuecomment-716601515)

Reply:
Thanks for the context @thehomebrewnerd !